From 1241c33f76f2708651f47c77a977e6d03d69d4ce Mon Sep 17 00:00:00 2001
From: yingbinx <yingbinx.zeng@intel.com>
Date: Wed, 9 Aug 2017 16:23:40 +0800
Subject: [PATCH 05/17] fix build warnings in storage

Updated the API of rpmb_ioc_cmd_set() to
fix build warning in storage proxy module.

Jira: None
Test: Device boots to home screen.
Change-Id: I3cb777047cb92b0e5c155afcc9b7180651b2d9a8
Signed-off-by: yingbinx <yingbinx.zeng@intel.com>
Signed-off-by: Huang, Yang <yang.huang@intel.com>
---
 trusty/storage/proxy/linux/rpmb.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trusty/storage/proxy/linux/rpmb.h b/trusty/storage/proxy/linux/rpmb.h
index 0def63b..f0c2e14 100644
--- a/trusty/storage/proxy/linux/rpmb.h
+++ b/trusty/storage/proxy/linux/rpmb.h
@@ -115,7 +115,7 @@ struct rpmb_ioc_cmd {
 #define rpmb_ioc_cmd_set(_cmd, _flags, _ptr, _n) do {         \
 	(_cmd).flags = (_flags);                              \
 	(_cmd).nframes = (_n);                                \
-	(_cmd).frames_ptr = (__aligned_u64)(intptr_t)(_ptr);  \
+	(_cmd).frames_ptr = (__u64)(intptr_t)(_ptr);  \
 } while (0)
 
 /**
-- 
2.7.4

